### ISSUE TITLE
Image in text uses topleft as anchor

### ISSUE THREAD

**Issue by hugolz** [Bowarc/chess_game#12]
Could be fixed by making a table / 2d list of the elements to know what are their pos 

obv make a new line when you find a \n char

**Comment by Bowarc** [comment]
Yes, but then you'll need to precompute the texts to know their dimensions before looping through them in draw
